lib/workers/global/index.ts

```diff
@@ -44,13 +46,38 @@ function haveReachedLimits(): boolean {
   return false;
 }
 
+/* istanbul ignore next */
+function checkEnv(): void {
+  const range = pkg.engines.node;
+  const rangeNext = pkg['engines-next']?.node;
+  if (process.release?.name !== 'node' || !process.versions?.node) {
+    logger.error(
```

Comment:
```suggestion
    logger.warn(
```

I suggest we make this warn, just in case there's something we've overlooked. Error will result in it exiting with a non-zero code